brevbaker/src/test/kotlin/no/nav/pensjon/brev/template/render/LetterRendererTest.kt

```diff
@@ -77,28 +80,28 @@ class LetterRendererTest {
     fun `render will pass each element of Content and letterScope to block`() {
         val expectedScope = letter.toScope()
         val expectedElements = listOf(
-            Element.OutlineContent.Paragraph(
+            createParagraph(
```

Comment:
Hva er grunnen til disse create*-funksjonene? Er det kun verbositet, eller er det andre grunner? Om det er fordi det er lettere å lese så kan vi jo også bare importere de respektive klassene og bruke konstruktørene. Altså:
```
Paragraph(
   listOf(
      Content(Literal(Bokmal to "hei")),
      Content(Literal(Bokmal to "jadda")),
   )
)
```

Reply:
Det er ein workaround for å kunne bruke `internal constructor` på desse klassane. Sia `create`-metodane ligg i `testFixtures` er dei ikkje tilgjengelege i prod-koden.